apps/workflow-server/pom.xml

```diff
@@ -27,7 +27,7 @@
       see https://github.com/translations-com/globallink-connect-cloud-api-java
     -->
     <gcc-restclient.version>3.1.3</gcc-restclient.version>
-    <cm.middle.core.version>2404.1</cm.middle.core.version>
+    <cm.middle.core.version>2406.0.1</cm.middle.core.version>
```

Comment:
So we are just using the latest currently existing AMP? Or should it be the first one, which would be 2406.0.0?

Reply:
We tested and approved for 2406.0.1 (2406.0 got re-released due to a security issue in a transitive dependency). Using 2406.0.1 seems to be the correct option here.

Still, as no relevant adaptations have been made to studio-client and other related code, the current state should also work for 2406.0.0.

Decision for now: Keep it at 2406.0.1.